receiver/splunkenterprisereceiver/metadata.yaml

```diff
@@ -6,4 +6,33 @@ status:
     development: [metrics]
   distributions:
   codeowners:
-    active: [shalper2, MovieStoreGuy]
\ No newline at end of file
+    active: [shalper2, MovieStoreGuy]
+
+attributes:
+  index.name:
+    description: The name of the index reporting a specific KPI
+    type: string
+  status:
+    description: The status message reported for a specific object
+    type: string
+  bucket.dir:
+    description: The bucket super-directory (home, cold, thawed) for each index
+    type: string
+
+metrics:
+  splunk.license.index.usage:
+    enabled: true
+    description: Gauge tracking the indexed license usage per index
+    unit: GBy
+    gauge:
+      value_type: double
+    attributes: [index.name]
+  # 'services/server/introspection/indexer'
+  splunk.server.introspection.indexer.throughput:
+    enabled: true
+    description: Gauge tracking average KBps throughput of indexer
+    unit: KBy
+    gauge:
+      value_type: double
+    # attribute `status` can be one of the following `normal`, `throttled`, `stopped`
+    attributes: [status]
```

Comment:
Name of the attribute should include namespaces according to the latest OTel spec recommendations. SO probably should be `splunk.indexer.status`?